server/database/load.json

```diff
@@ -1,2543 +1,2768 @@
 {
     "core-model-1-4:control-construct": {
```

Comment:
can we remove this file and openapi.yaml file as part of this pull request because these files are already part of pull request 166 and there are no additional changes in this pull request

Reply:
Removed openapi.yaml